spec/support/feature_steps.rb

```diff
@@ -209,6 +204,8 @@ def fill_claim
 
   def fill_et_claim
     fill_in 'claim_et_identifier', with: '12345'
+    # for some reason this has to be clicked twice.
+    click_button 'Continue'
```

Comment:
@colinbruce I find the reason for this. For some reason he is on "http://www.example.com/questions/probate?locale=en" url instead of "http://www.example.com/questions/claim?locale=en" I will fix it.